var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -279,13 +281,18 @@ def configure_args(self):
             configure_args.extend(["--without-hip", "--disable-rocrand", "--disable-rocsparse"])
 
         if "+sycl" in spec:
-            configure_args.append("--with-sycl")
             sycl_compatible_compilers = ["dpcpp", "icpx"]
             if not (os.path.basename(self.compiler.cxx) in sycl_compatible_compilers):
                 raise InstallError(
                     "Hypre's SYCL GPU Backend requires DPC++ (dpcpp)"
                     + " or the oneAPI CXX (icpx) compiler."
                 )
+            sycl_inc = ""
+            sycl_pkgs = ["intel-oneapi-dpl", "intel-oneapi-mkl"]
+            for pkg in sycl_pkgs:
+                if "^" + pkg in spec:
+                    sycl_inc += spec[pkg].headers.include_flags + " "
```

Comment:
It seems to me both dependencies are enforced when we activate `+sycl`, so this:
```suggestion
            for pkg in ("intel-oneapi-dpl", "intel-oneapi-mkl")
                sycl_inc += spec[pkg].headers.include_flags + " "
```
should be enough, right?

Reply:
Is the PR still a work in progress?